pkg/occlient/occlient.go

```diff
@@ -2684,6 +2671,7 @@ func tar(tw *taro.Writer, fileName string, destFile string) error {
 
 // recursiveTar function is copied from https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/cp.go#L319
 func recursiveTar(srcBase, srcFile, destBase, destFile string, tw *taro.Writer) error {
+	glog.V(4).Infof("recursiveTar, srcBase: %s, srcFile: %s, destBase: %s, destFile: %s", srcBase, srcFile, destBase, destFile)
```

Comment:
again, maybe a little bit more info here.

I also noticed that this message has a different style than the similar message in `makeTar` function.

`makeTar` one uses `makeTar: ` this one is `recursiveTar,` (function name followed by `,` vs `:`)

Reply:
Ack I'll quickly fix it :)